circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/ReplicaTableFactory.java

```diff
@@ -46,19 +47,20 @@
 public class ReplicaTableFactory {
 
   /**
-   * Parameter to indicate we wish to retain table statistics >= Hive 2.0.0. See <a href=
-   * "https://github.com/apache/hive/blob/release-2.0.0/common/src/java/org/apache/hadoop/hive/common/StatsSetupConst.java#L120">
-   * hive-exec:2.0.0#StatsSetupConst:120</a>.
+   * Parameter to indicate we wish to retain table statistics >= Hive 2.0.0.
    */
-  private static final String DO_NOT_UPDATE_STATS = "DO_NOT_UPDATE_STATS";
+  private static final String DO_NOT_UPDATE_STATS = StatsSetupConst.DO_NOT_UPDATE_STATS;
 
   /**
-   * Parameter to indicate we wish to retain table statistics < Hive 2.0.0. See <a href=
-   * "https://github.com/apache/hive/blob/release-1.0.0/common/src/java/org/apache/hadoop/hive/common/StatsSetupConst.java#L134">
-   * hive-exec:1.0.0#StatsSetupConst:134</a>.
+   * Parameter to indicate we wish to retain table statistics <= Hive 2.0.1.
    */
   private static final String STATS_GENERATED_VIA_STATS_TASK = "STATS_GENERATED_VIA_STATS_TASK";
 
+  /**
+   * Parameter to indicate we wish to retain table statistics >= Hive 2.1.0.
+   */
+  private static final String STATS_GENERATED = "STATS_GENERATED";
```

Comment:
I can't remember why this is like it is but looks like `StatsSetupConst.` is missing (?)